pkg/daemon/daemon.go

```diff
@@ -164,6 +164,10 @@ func (d *Daemon) sharedErofsMount() error {
 		return err
 	}
 
+	if err := os.MkdirAll(d.FscacheWorkDir(), 0755); err != nil {
```

Comment:
Maybe we can ensure this directory on snapshotter startup?

Reply:
This dir seems pre-image , and is generated by SnapshotDir and SnapshotID, should we use a fixed directory?

Reply:
Ok, indeed it's per image, sorry I miss that.